boto3/dynamodb/types.py

```diff
@@ -62,6 +62,9 @@ def __repr__(self):
         return 'Binary(%r)' % self.value
 
     def __str__(self):
+        return self.value if six.PY2 else self.value.decode('utf-8')
```

Comment:
In PY2 the decode would return a unicode string, not a regular `str`. My thinking was that it would be odd to return unicode from `__str__`. On the other side, it would be acceptable to return bytes, since byes and str are the same thing (only in PY2, of course).